docs/source/cdk/cdk-deployment.rst

```diff
@@ -0,0 +1,30 @@
+CDK deployment
+==============
+Deploy
+~~~~~~
+You will need to make a **copy** of app_template_dev.py file with a different name (app__dev.py) and keep a copy of
```

Comment:
```suggestion
~~~~~~
You will need to make a **copy** of :file:`app_template_dev.py` file with a different name :file:`app_<initials>_dev.py` and keep a copy of
```
Thanks for copying this over into the docs! See here for an update of what this should be to avoid the links: https://github.com/IMAP-Science-Operations-Center/sds-data-manager/pull/111
In rst I think you want the file directive to format it indicating it is an external file location.
https://devguide.python.org/documentation/markup/